README.md

```diff
@@ -1,3 +1,112 @@
-# tdr-terraform-environments
+# TDR Terraform Environments
 
-Readme for tdr-terraform-environments repository
+This respository contains the Terraform code to create the AWS resources needed to support the TDR application.
+
+## Terraform Structure
+
+The prototype is divided into separate Terraform modules that represent the different AWS resources that are needed to for the TDR project.
+
+The different modules are used by Terraform workspaces which represent three AWS environments:
+
+* intg
+* staging
+* prod
+
+## Getting Started
+
+### Terraform Backend
+
+Ensure that the Terraform backend has been created.
+
+See here: https://github.com/nationalarchives/tdr-dev-documentation/blob/tkAWSInfrastructureSetup/tdr-create-aws-instructure-setup.md
+
+### Install Terraform locally
+
+See: https://learn.hashicorp.com/terraform/getting-started/install.html
+
+### Install AWS CLI Locally
+
+See: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
+
+### Install Terraform Plugins on Intellij
+
+HCL Language Support: https://plugins.jetbrains.com/plugin/7808-hashicorp-terraform--hcl-language-support
+
+### Add AWS Credentials and Profiles
+
+1. Update local AWS credentials file (~/.aws/credentials) with Terraform user credentials for the TDR AWS management account:
```

Comment:
Just to clarify - should you use credentials for your own IAM user within the TDR AWS management account? It's not asking you to save credentials for some Terraform user?

Also, what's the long-term goal for how we're going to run environment-specific Terraform? Will it happen in Jenkins or in ECS, rather than a dev machine? Are we going to restrict it so that developers _can't_ run Terraform with their dev IAM accounts?

Reply:
Amended the wording to make a bit clearer.

The long term goal will be to run via Jenkins, and not allow developers to run from their local machines.